Ports/open-adventure/patches/ReadMe.md

```diff
@@ -0,0 +1,11 @@
+# Patches for open-adventure on SerenityOS
+
+## `0001-Use-libeditline-instead-of-libedit.patch`
+
+Three changes:
+
+1. Change `libedit` to [`editline`](../../editline/) in the `Makefile`.
+
+2. Change `#include <editline/readline.h>` to `#include <editline.h>`.
+
+3. Move `stdio.h` before `editline.h` because it uses `FILE`.
```

Comment:
This does not match the commit message in the actual patch. Is this file properly autogenerated?

Reply:
No. I wrote that. I didn't know it was supposed to be autogenerated. I don't see this convention being mentioned anywhere in the docs. I'll change it.

Reply:
Done.